playbooks/roles/lets-encrypt/templates/response.yaml.j2

```diff
@@ -0,0 +1,13 @@
+"acme-enter-email": "{{ streisand_admin_email }}"
+"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf": true
+"acmetool-quickstart-choose-server": https://acme-v01.api.letsencrypt.org/directory
+"acmetool-quickstart-choose-method": redirector
+# This is only used if "acmetool-quickstart-choose-method" is "webroot".
+"acmetool-quickstart-webroot-path": "/var/www/foo/bar/.well-known/acme-challenge"
+"acmetool-quickstart-complete": true
+"acmetool-quickstart-install-cronjob": true
+"acmetool-quickstart-install-haproxy-script": true
```

Comment:
I think this can be false - we don't install `haproxy`.

Reply:
@cpu Oh yes. I'll fix that.